mitodl/micromasters
# Added person search

#### What are the relevant tickets?
Part of #2479

#### What's this PR do?
Adds a text search field which searches based on first, last, and preferred names for people. The index has also been updated to add an analyzed version of these fields which filters out accented characters

#### How should this be manually tested?
Run `./manage.py recreate_index` then go to the learners page and search with the search box. The search should execute soon after you type a character
